source/chapters/microphones.rst

```diff
@@ -411,3 +411,27 @@ input.
 
 DJ controllers typically do not supply phantom power required for condenser
 microphones. Stand-alone audio interfaces typically do supply phantom power.
+
+Microphone Ducking
+------------------
+While using Mixxx, you might want to talk through your microphone as the music plays, therefore, you might need the music to automatically fade into the background each time you speak, or you might prefer lowering the volume of the music to a specific level. Microphone ducking helps you achieve this.
```

Comment:
The word "might" appears a lot here.  We can simplify:

"While using Mixxx with a microphone, you might want the music to automatically fade into the background each time you speak, or you might prefer lowering the volume of the music to a specific level. Microphone ducking helps you achieve this. The music will 'duck' out of the way of the microphone audio."

Reply:
(I think it's worth explaining the word "duck" just in case people are having trouble with that)

Reply:
Thank you. I agree. 
I have added a definition for ducking just before this paragraph on Line 417. You can read through and let me know if where it might need correction....